custom/git-app-deployer/main.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/litmuschaos/test-tools/pkg/log"
+	"github.com/openebs/maya/pkg/util/retry"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+func main() {
+
+	var kubeconfig *string
+	var timeout int
+	file := flag.String("file", "", "absolute path of the sock-shop")
+	flag.Parse()
+
+	if *file == "" {
```

Comment:
can you please comment on this? Why did we need different values of timeout based on the file name?